build.zig

```diff
@@ -101,9 +101,20 @@ pub fn build(b: *std.Build) !void {
         }),
     });
     example.root_module.addImport("z3", z3_bindings);
-
+    b.installArtifact(example);
     const run_example = b.step("example", "Runs the example");
     run_example.dependOn(&b.addRunArtifact(example).step);
+
+    const tests = b.addTest(.{ .root_module = b.createModule(.{
+        .root_source_file = b.path("src/tests.zig"),
+        .optimize = optimize,
+        .target = target,
+        .imports = &.{.{ .name = "z3", .module = z3_bindings }},
+    }) });
+    b.installArtifact(tests);
+    tests.filters = if (b.option([]const u8, "test-filter", "test filter")) |f| b.dupeStrings(&.{f}) else &.{};
```

Comment:
```suggestion
    const tests = b.addTest(.{ .root_module = b.createModule(.{
        .root_source_file = b.path("src/tests.zig"),
        .optimize = optimize,
        .target = target,
        .imports = &.{.{ .name = "z3", .module = z3_bindings }},
        .filters = b.option([]const []const u8), "test-filter", "") orelse &.{},
    }) });
    b.installArtifact(tests);
```

or something like that